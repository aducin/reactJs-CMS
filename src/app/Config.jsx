import React from 'react';

const Config = {
	accountColumns: [
		{ id: 1, name: 'Lp.', value: null },
		{ id: 2, name: 'Klient', value: 'recipient' },
		{ id: 3, name: 'Numer nadania', value: 'address' },
		{ id: 4, name: 'Kwota', value: 'floatAmount' },
		{ id: 5, name: 'Rodzaj', value: 'type' },
		{ id: 6, name: 'Nr paragonu', value: 'receipt' },
		{ id: 7, name: 'Data paragonu', value: 'receiptTimestamp' },
		{ id: 8, name: 'Data wpłaty', value: 'cashTimestamp' },
		{ id: 9, name: 'Lok.', value: 'locs' },
		{ id: 10, name: 'Wagony', value: 'coach' },
		{ id: 11, name: 'Element', value: 'element' },
		{ id: 12, name: 'Części', value: 'accessories' },
		{ id: 13, name: 'Książki', value: 'book' },
		{ id: 14, name: 'Auta', value: 'car' },
		{ id: 15, name: 'Uwagi', value: 'remarks' }
	],
	accountNumbers: ['receipt', 'amount', 'locs', 'coach', 'element', 'accessories', 'book', 'car'],
	accountObligatory: ['recipient', 'amount', 'closed', 'type'],
	accountStates: [
		{ id: 0, name: 'Otwarty' },
		{ id: 1, name: 'Zamknięty' },
		{ id: 2, name: 'Nieodebrany' }
	],
	accountTypes: [
		{ id: 1, name: 'Przelew' },
		{ id: 2, name: 'Pobranie' },
		{ id: 3, name: 'Usł. inf.' },
		{ id: 4, name: 'Sprzedaż bezp.' }
	],
	active: [
		{
			id: 0,
		    name: 'Nieaktywny'
		},
		{
			id: 1,
		    name: 'Aktywny'
		}
	],
	ajaxConfig: {
		  		headers: {'X-My-Custom-Header': 'Header-Value'}
			},
	condition: [
		{
			id: 'new',
		    name: 'Nowy'
		},
		{
			id: 'used',
		    name: 'Używany'
		},
		{
			id: 'renewed',
		    name: 'Odnowiony'
		}
	],
	css: {
		centered: {
			textAlign: 'center'
		},
	},
	fields: [
		{
			link: 'products', 
			name: 'Produkty'
		},
		{
			link: 'orders',
			name: 'Zamówienia'
		},
		{
			link: 'postal',
			name: 'Wysyłki'
		},
		{
			link: 'accounts',
			name: 'Rachunki'
		},
		{
			link: 'customers',
			name: 'Klienci'
		}
	],
	images: {
		imgCss: {
		    height: '100px',
		   	width: '100px',
			color: 'blue',
			border: '1px solid #ddd',
			borderRadius: 3,
		},
		imgCssSmall: {
		    height: '65px',
		    width: '65px',
			color: 'blue',
			border: '1px solid #ddd',
			borderRadius: 3,
		},
		imgCssFixed: {
			position: 'fixed',
			left: '2.5%',
			top: '33%',
			maxWidth: '250px',
			maxHeight: '250px',
			border: '2px solid #ddd',
			borderRadius: 5,
		},
	},
	message: {
		account: {
			accessories: 'Części zamienne',
			add: 'Dodaj rachunek',
			addTitle: 'Dodaj nowy rachunek',
			amount: 'Kwota',
			automatic: 'Lista rachunków - ostatnie 10 wyników',
			cars: 'Auta',
			cashDate: 'Data wpłaty',
			coaches: 'Wagony',
			closed: 'Rachunek zamknięty - edycja zablokowana!',
			createXml: 'Wygeneruj plik',
			dateFrom: 'Data początkowa',
			dateTo: 'Data końcowa',
			defaultOption: { id: -1, name: 'Wybierz'},
			downloadXml: 'Kliknij tutaj aby pobrać',
			elements: 'Elementy makiety',
			filter: 'Wyszukiwanie zaawansowane',
			literature: 'Literatura',
			locs: 'Lokomotywy',
			managing: 'Zarządzanie rachunkami',
			modify: 'Edytuj rachunek',
			modifyTitle: 'Zmodyfikuj istniejący rachunek',
			maxAmount: ' (maksymalna jednorazowa ilość - uściślij kryteria)',
			notAutomatic: ['Lista rachunków z podanego zakresu: ', 'wyników'],
			order: 'Zamówienie',
			receipt: 'Numer dokumentu',
			receiptDate: 'Data paragonu',
			remarks: 'Uwagi',
			shipment: 'Numer przesyłki',
			state: 'Stan rachunku',
			type: 'Typ rachunku'
		},
		actions: {
			clear: 'Wyczyść wyniki',
			goBackToList: 'Powrót do listy',
			history: 'Historia zmian',
			save: 'Zapisz',
			self: 'Akcje:',
			showList: 'Kliknij aby rozwinąć listę',
			showMiniatures: 'Kliknij aby wyświetlić miniatury'
		},
		additional: {
			deletePhotos: 'Usuń zdjęcia',
			modify: 'Zapisz produkt jako podmieniony'
		},
		alreadyChecked: 'Podane żądanie zostało już zrealizowane',
		authorisation: 'Proszę czekać - trwa weryfikacja...',
		categoryAmount: 'Ilość aktywnych kategorii: ',
		close: 'Zamknij',
		condition: 'Stan:',
		currency: "zł",
		customer: {
			address: 'Adres e-mail',
			addresses: 'Zarejestrowane adresy:',
			delete: 'Usuń konto',
			deleteConfirm: 'Czy na pewno chcesz usunąć bieżące konto?',
			detailsIndicator: 'Szczegóły',
			details: 'Dane dotyczące klienta o adresie: ',
			mailDetails: 'Mail (szczegóły)',
			mailTransfer: 'Mail (transfer)',
			noAccount: {
				new: 'Brak konta w nowym sklepie.',
				old: 'Brak konta w starym sklepie.'
			},
			noAddress: 'Klient nie podał jeszcze żadnego adresu.',
			noOrder: 'Klient nie złożył jeszcze zamówienia',
			orders: 'Złożone zamówienia: ',
			panels: {
				new: 'Nowy panel',
				old: 'Stary Panel'
			},
			typeAnAddress: 'Podaj adres email, aby wyszukać klienta.'
		},
		delete: 'Usuń',
		differentPrices: 'Różnica cen! W nowym panelu cena to: ',
		differentQuantities: 'Różnica ilości! W nowym panelu ilośc to: ',
		discount: {
			new: "Rabat w nowym panelu to: ",
			old: "Rabat w starym panelu to: "
		},
		discountNet: "Cena nominalna: ",
		edition: 'Edycja',
		editionSearch: "Wczytywanie danych produktu - proszę czekać...",
		emptyField: 'Puste pole z nazwą produktu',
		error: "Wystąpił nieoczekiwany błąd...",
		fullEdition: "Pełna edycja produktu ID: ",
		goBack: 'Powrót',
		historyTitle: "Historia zmian produktu ID: ",
		labels: {
			active: 'Aktywność:',
			amount: 'Kwota rachunku',
			categories: 'Kategorie:',
			condition: 'Stan:',
			date: 'Podaj datę',
			description: {
				nameFull: 'Pełny opis:',
				nameShort: 'Krótki opis:',
				placeholder: 'Podaj opis'
			},
			email: 'Podaj adres email',
			filename: 'Nazwa pliku',
			linkRewrite: {
				name: 'Link:',
				placeholder: 'Podaj adres URL'
			},
			manufactorer: 'Producent:',
			metaDescription: {
				name: 'Meta-opis produktu:',
				placeholder: 'Podaj opis'
			},
			metaTitle: {
				name: 'Meta-tytuł produktu:',
				placeholder: 'Podaj opis'
			},
			name: {
				name: 'Nazwa:',
				placeholder: 'Podaj nazwę'
			},
			order: 'Numer zamówienia',
			price: {
				nameNew: 'Cena (NP):',
				nameOld: 'Cena (SP):',
				placeholder: 'Podaj cenę'
			},
			printings: 'Dokumenty do wydruku:',
			productName: 'Nazwa produktu',
			quantity: {
				name: 'Ilość:',
				placeholder: 'Podaj ilość'
			},
			receipt: 'Podaj numer',
			remarks: 'Wpisz uwagi',
			shipment: 'Numer przesyłki',
			tags: {
				name: 'Tagi produktu:',
				placeholder: 'Podaj po przecinku'
			},
		},
		lastOrdersSearch: "Sprawdzanie ostatnich zamówień - proszę czekać...",
		lastSearchList: "Lista z poprzedniego wyszukiwania: ",
		listOfProducts: "Lista wyników: ",
		loading: 'Proszę czekać - dane w trakcie ładowania...',
		logout: 'Zostałeś skutecznie wylogowany',
		min3characters: 'Podaj przynajmniej 4 znaki',
		modifiedSearch: 'Ładowanie produktów podmienionych - proszę czekać...',
		nameCategory: 'Dowolna kategoria',
		nameManufactorer: 'Dowony producent',
		no: 'Nie',
		noLastOrder: 'Obecnie brak nowych zamówień',
		noModified: 'Obecnie brak produktów podmienionych',
		noNameList: "Brak produktów spełniających podane kryteria",
		noPrintings: 'Obecnie brak dokumentow do wydruku.',
		orders: {
			back: 'Powrót do zamówienia',
			chooseAction: 'Wybierz akcję',
			choosePanel: 'Wybierz zamówienie',
			defaultShipmentNumber: '(00)',
			delete: 'Usuń wyniki',
			details: 'Szczegóły zamówienia nr ',
			email: 'Adres e-mail:',
			even: 'Wyrównaj ilości',
			evenTitle: 'Wyrównanie ilości produktów - zamówienie nr ',
			find: 'Wyszukaj',
			fullEdition: 'Pełna edycja',
			newOrder: {
				new: 'Nowe zamówienie - nowy panel',
				old: 'Nowe zamówienie - stary panel'
			},
			noAction: 'Podaj numer zamówienia i wybierz akcję.',
			noVoucher: 'Klient nie posiada zamówień na min. 50zł!',
			sameAmount: 'Równa ilość...',
			send: 'Wyślij maila',
			shipmentNumber: 'Prześlij numer przesyłki',
			showEmail: 'Zobacz treść maila',
			summary: 'PODSUMOWANIE:',
			total: 'Suma wszystkich produktów:',
			totalDiscount: 'Suma po rabacie:',
			totalShipping: 'Suma produktów z wysyłką:',
			totalShippingDiscount: 'Suma z wysyłką po rabacie:',
			typeShipmentNumber: { name: 'Wprowadź numer przesyłki', placeholder: 'Podaj opis' },
			voucherLast: 'Numer ostatniego kuponu:',
			voucherMax: 'Maksymalny numer kuponu to 5!',
			voucherMin: 'Minimalny numer kuponu to 1!',
			voucherMinus: 'Odejmij kupon',
			voucherNumber: 'Numer kuponu',
			voucherPlus: 'Dodaj kupon',
			voucherTitle: 'Wyniki wyszukiwania kuponów dla Klienta: '
		},
		otherManufactorer: {
			id: 0,
			name: 'Brak (inny producent)'
		},
		pleaseChoose: {
			id: 0, 
			name: 'Proszę wybrać'
		},
		postal: {
			add: 'Dodaj kwotę',
			afterChange: 'Kwota po zmianie',
			currentAmount: 'Bieżąca kwota:',
			history: 'Historia zmian (ostatnie 5 wpisów):',
			inProgress: 'Proszę czekać - operacja w toku...',
			modalAdd: 'Dodaj środki',
			modalSubtract: 'Odejmij środki',
			saveAmount: 'Zapisz kwotę',
			subtract: 'Odejmij kwotę',
			toChange: 'Kwota do zmiany',
			typeAmount: 'Podaj kwotę'
		},
		priceDisabled: 'Zmiana cen wyłącznie przez pełną edycję!',
		printingsSearch: 'Ładowanie dokumentów do wydruku - proszę czekać...',
		products: {
			chooseFile: 'Dodaj dokument',
			description: 'Opis dokumentu',
			idSearch: 'Wyszukiwanie po ID:',
			oneFileOnly: 'Proszę dodawać pliki pojedynczo',
			nameSearch: 'Wyszukiwanie po nazwie:',
			newPrinting: 'Nowy dokument do wydruku',
			printingsLoading: 'Dokumenty są ładowane - proszę czekać...',
			saveFile: 'Zapisz dokument'
		},
		quantity: {
			equal: 'Ta sama ilość w obu bazach.',
			notEqual: 'UWAGA! Ilość w starej bazie to: '
		},
		realPrice: {
			new: ' Realna cena w nowym panelu to: ',
			old: ' Realna cena w starym panelu to: ',
			suffix: ' - rabat wynosi: ',
		},
		simpleEdition: 'Skrócona edycja produktu ID: ',
		singleEditionAnother: 'Dane kolejnego produktu są ładowane...',
		title: {
			accounts: 'Manager rachunków Ad9BIS',
			customers: 'Manager klientów Ad9BIS',
			orders: 'Manager zamówień Ad9BIS',
			postal: 'Manager wysyłek Ad9BIS',
			products: 'Manager produktów Ad9BIS'
		},
		today: 'Dzisiaj'
	},
	orderActions: [
		{
			id: 3,
			action: 'voucher',
			name: 'Sprawdź kupon'
		},
		{
			id: 4,
			action: 'discount',
			name: 'Oblicz 15% rabat'
		},
		{
			id: 5,
			action: 'remindNew',
			name: 'Wyślij ponowny mail (NP)'
		},
		{
			id: 6,
			action: 'remindOld',
			name: 'Wyślij ponowny mail (SP)'
		}
	],
	orderPanels: [
		{
			id: 1, 
			action: 'orderNew',
			name: 'Nowy panel'
		},
		{
			id: 2, 
			action: 'orderOld',
			name: 'Stary panel'
		}
	],
	timer: 3000,
	url: {
		filePath: 'http://modele-ad9bis.pl/cms_spa/files/',
		path: "http://modele-ad9bis.pl/",
		pathAccount: "accounts",
		pathCustomers: "customers",
		pathOrder: "orders",
		pathPostal: "postal",
		pathProducts: "products",
		pathSuffix: "reactjs-training/dist/#/",
		serverPath: "http://modele-ad9bis.pl/cms_spa/web/",
		shopUrl: 'http://ad9bis.vot.pl/wagony-towarowe/',
	}
}


export default Config