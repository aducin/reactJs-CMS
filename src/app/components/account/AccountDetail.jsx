import React from 'react';

import 'font-awesome/css/font-awesome.min.css';

import Title from '../dumb/Title.jsx';

const leftMargin = {marginLeft: '20px'};

const accountDetail = ( props ) => {
  if (props.empty) {
    return (
      <Title title={props.message.account.noData} />
    )
  } else {
    let field = props.sortBy;
    const sortFn = (a, b) => {
      if (a[field] === null) {
        return -1;
      } else {
        if (typeof(a[field]) === 'string') {
          var nameA = a[field].toLowerCase().localeCompare(b[field].toLowerCase(), "pl-PL");
          var nameB = b[field].toLowerCase().localeCompare(a[field].toLowerCase(), "pl-PL");
          if (nameA < nameB)
            return -1
          if (nameA > nameB)
            return 1
          return 0;
        } else if (typeof(a[field]) === 'number') {
          return a[field] - b[field];
        }
      }
    };
    let content, head, summary, row3, row17;
    let contentArray = [];
    let list = props.data.list.map((el) => {
      let curType = props.types.filter((secondEl) => { return parseInt(secondEl.id) === parseInt(el.type); });
      if (curType[0]) {
        el.typeName = curType[0].name;
      }
      return el;
    });
    list.sort(sortFn);
    if (!props.ascending) list.reverse();
    let message = props.message;
    let title = props.data.automatic ? message.account.automatic : message.account.notAutomatic[0] + list.length + ' ' + message.account.notAutomatic[1];
    if (!props.data.automatic && props.data.amount === props.data.maxAmount) {
      title += message.account.maxAmount;
    }
    let headArray = props.columns.map((el, index) => {
      if (el.value) {
        let icon;
        if (el.value !== field) {
          icon = 'fa fa-sort';
        } else {
          if (props.ascending) {
            icon = 'fa fa-sort-desc';
          } else {
            icon = 'fa fa-sort-asc';
          }
        }
        return (
          <th onClick={() => props.sortTable(el.value, el.sort)} key={index} class="textAlignCenter cursorPointer">{el.name}<i class={icon} style={leftMargin} aria-hidden="true"></i></th>
        );
      } else {
        return (
          <th key={index} class="textAlignCenter">{el.name}</th>
        );
      }
    });
    list.forEach((el, index) => {
      let active = el.id === props.selectedRow;
      let activeClass = active ? 'selected cursorPointer textAlignCenter' : 'cursorPointer textAlignCenter';
      let closedClass = el.closed ? 'colorWarning' : 'colorSuccess';
      let remarks = el.remarks ? el.remarks : '---';
      contentArray.push(
        <tr key={ index } onClick={ () => props.selectRow(el.id)} class={activeClass}>
          <td class={closedClass}>{index + 1}.</td>
          <td>{el.recipient}</td>
          <td>{el.address}</td>
          <td>{el.amount}{message.currency}</td>
          <td>{el.typeName}</td>
          <td>{el.receipt}</td>
          <td>{el.receiptTime}</td>
          <td>{el.cashTime}</td>
          <td>{el.locs}</td>
          <td>{el.coach}</td>
          <td>{el.element}</td>
          <td>{el.accessories}</td>
          <td>{el.book}</td>
          <td>{el.car}</td>
          <td>{remarks}</td>
        </tr>
      );
    });
    let numbers = props.data.amounts;
    if (numbers.amount3 > 0) {
      row3 = (
        <tr>
          <td key="1" class="textAlignCenter">3%</td>
          <td key="2" class="textAlignCenter">{ numbers.amount3.toFixed(2) }{ message.currency }</td>
          <td key="3" class="textAlignCenter">{ numbers.tax3.toFixed(2) }{ message.currency }</td>
        </tr>
      );
    }
    if (numbers.amount17 > 0) {
      row17 = (
        <tr>
          <td key="1" class="textAlignCenter">17%</td>
          <td key="2" class="textAlignCenter">{ numbers.amount17.toFixed(2) }{ message.currency }</td>
          <td key="3" class="textAlignCenter">{ numbers.tax17.toFixed(2) }{ message.currency }</td>
        </tr>
      );
    }
    if (numbers.amount3 > 0 || numbers.amount17 > 0) {
      summary = (
        <div class="col-xs-12 col-md-4 pull-left">
          <table class="table table-striped table-bordered">
            <thead>
            <tr>
              <th key="1" class="textAlignCenter">{ message.account.summary }</th>
              <th key="2" class="textAlignCenter">{ message.account.summaryAmount }</th>
              <th key="3" class="textAlignCenter">{ message.account.summaryTax }</th>
            </tr>
            </thead>
            <tbody>
            { row3 }
            { row17 }
            </tbody>
          </table>
        </div>
      );
    }
    head = (
      <thead>
      <tr>
        {headArray}
      </tr>
      </thead>
    );
    content = (
      <tbody>
      { contentArray }
      </tbody>
    );
    return(
      <div class="col-xs-12 pull-left bgrContent borderRadius10 marginTop40px">
        <div class="col-xs-12">
          <Title title={title} />
        </div>
        <div class="marginTop20px">
          <table class="table table-striped table-bordered table-horizontal">
            {head}
            {content}
          </table>
          <div class="col-xs-12 col-md-4 pull-left"></div>
          {summary}
          <div class="col-xs-12 col-md-4 pull-left"></div>
        </div>
      </div>
    );
  }
}

export default accountDetail;