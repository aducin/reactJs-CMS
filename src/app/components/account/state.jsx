const state = {
  ajaxSent: false,
  ascending: false,
  createXml: false,
  innerFields: ['amount', 'automatic', 'list', 'maxAmount', 'message'],
  inProgress: false,
  link: undefined,
  modal: false,
  modalDisable: false,
  modalMessage: {
    text: null,
    type: null
  },
  modalObj: {},
  selected: {
    type: -1,
    state: -1,
    dateFrom: null,
    dateTo: null
  },
  modalObjError: {},
  selectedRow: null,
  sortBy: 'id'
}

export default state;