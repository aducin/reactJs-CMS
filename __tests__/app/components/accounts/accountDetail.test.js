import React from 'react';
import TestRenderer from 'react-test-renderer';

import Config from '../../../../src/app/Config';
import AccountDetail from '../../../../src/app/components/account/AccountDetail.jsx';
import state from '../../../../src/app/helper/accountState';

const ascending = true;
const data = {
  amounts: {
    amount3: 10.99,
    amount17: 0,
    tax3: 3.88,
    tax17: 0
  },
  automatic: true,
  success: true,
  list: [
    {
      id: 1000,
      amount: 10.99,
      recipient: 'PayU',
      address: null,
      receipt: 9834,
      locs: null,
      coach: null,
      element: 10,
      accessories: 5,
      book: null,
      car: null,
      closed: 1,
      remarks: null,
      createTime: "2018-06-21 00:00:00",
      cashTime: "2018-06-21",
      receiptTime: "2018-06-22"
    }
  ]
};
const empty = false;
const mockFunction = () => { return false; };
const selectedRow = 1;
const sortBy = 'id';

const details = TestRenderer.create(
  <AccountDetail
    ascending={ascending}
    data={data}
    empty={empty}
    selectedRow={selectedRow}
    selectRow={mockFunction}
    sortTable={mockFunction}
    sortBy={sortBy}
  />
);

it('AccountDetails renders correctly', async () => {
  const tree = details.toJSON();
  expect(tree).toMatchSnapshot();
});
