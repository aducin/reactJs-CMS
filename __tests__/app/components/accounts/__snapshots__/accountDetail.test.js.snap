// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountDetails renders correctly 1`] = `
<div
  className="col-xs-12 pull-left bgrContent borderRadius10 marginTop40px"
>
  <div
    className="col-xs-12"
  >
    <div
      className="container"
    >
      <div
        className="col-xs-12"
        style={null}
      >
        <h3>
          Lista rachunków - ostatnie 10 wyników
        </h3>
      </div>
    </div>
  </div>
  <div
    className="marginTop20px"
  >
    <table
      className="table table-striped table-bordered table-horizontal"
    >
      <thead>
        <tr>
          <th
            className="textAlignCenter"
          >
            Lp.
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Klient
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Numer nadania
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Kwota
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Rodzaj
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Nr paragonu
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Data paragonu
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Data wpłaty
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Lok.
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Wagony
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Element
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Części
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Książki
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Auta
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
          <th
            className="textAlignCenter cursorPointer"
            onClick={[Function]}
          >
            Uwagi
            <i
              aria-hidden="true"
              className="fa fa-sort"
              style={
                Object {
                  "marginLeft": "20px",
                }
              }
            />
          </th>
        </tr>
      </thead>
      <tbody>
        <tr
          className="cursorPointer textAlignCenter"
          onClick={[Function]}
        >
          <td
            className="colorWarning"
          >
            1
            .
          </td>
          <td>
            PayU
          </td>
          <td />
          <td>
            10.99
            zł
          </td>
          <td />
          <td>
            9834
          </td>
          <td>
            2018-06-22
          </td>
          <td>
            2018-06-21
          </td>
          <td />
          <td />
          <td>
            10
          </td>
          <td>
            5
          </td>
          <td />
          <td />
          <td>
            ---
          </td>
        </tr>
      </tbody>
    </table>
    <div
      className="col-xs-12 col-md-4 pull-left"
    />
    <div
      className="col-xs-12 col-md-4 pull-left"
    >
      <table
        className="table table-striped table-bordered"
      >
        <thead>
          <tr>
            <th
              className="textAlignCenter"
            >
              Podsumowanie - stawka
            </th>
            <th
              className="textAlignCenter"
            >
              Kwota - ogółem
            </th>
            <th
              className="textAlignCenter"
            >
              Należny podate
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td
              className="textAlignCenter"
            >
              3%
            </td>
            <td
              className="textAlignCenter"
            >
              10.99
              zł
            </td>
            <td
              className="textAlignCenter"
            >
              3.88
              zł
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <div
      className="col-xs-12 col-md-4 pull-left"
    />
  </div>
</div>
`;
